types/api/v1beta1/event.proto

```diff
@@ -0,0 +1,162 @@
+syntax = "proto3";
+
+option go_package = "github.co/aquasecurity/tracee/api/v1beta1";
+
+package tracee.v1beta1;
+
+message Event {
+    int64 timestamp = 1;
+    int64 id = 2;
+    string name = 3;
+    // TODO: version? are we going to have version here, or just at the definition?
```

Comment:
Version is static information about the event, so at least for now, just in the definition.